reactor-netty5-http/src/test/java/reactor/netty5/http/client/HttpClientProxyTest.java

```diff
@@ -90,7 +90,7 @@ void proxy_1(Hoverfly hoverfly) {
 				            true))
 				    .expectNextMatches(t ->
 				            t.getT2().contains("Hoverfly") &&
-				                "FOUND".equals(t.getT2().get("Logging-Handler")) &&
+				                "FOUND".equals(t.getT2().get("Logging-Handler").toString()) &&
```

Comment:
This may produce `NullPointerException`

Reply:
I have added a "getHeader" helpers method in BaseHttpTest.java, which avoids to do the toString() everywhere.
There are many other places where toString() was used, so I also fixed with the getHeader helper, please check.